packages/auth-client/src/storage.ts

```diff
@@ -70,6 +70,9 @@ export class LocalStorage implements AuthClientStorage {
  * @see implements {@link AuthClientStorage}
  */
 export class IdbStorage implements AuthClientStorage {
+  constructor(private options?: DBCreateOptions) {
```

Comment:
```suggestion
  #options: DBCreateOptions;

  /**
   * @param options - DBCreateOptions
   * @param options.dbName - name for the indexeddb database
   * @param options.storeName - name for the indexeddb Data Store
   * @param options.version - version of the database. Increment to safely upgrade
   * @constructs an {@link IdbStorage}
   * @example
   * ```typescript
   * const storage = new IdbStorage({ dbName: 'my-db', storeName: 'my-store', version: 2 });
   * ```
   */
  constructor(options?: DBCreateOptions) {
    this.#options = options ?? {};
```